index.js

```diff
@@ -39,7 +45,19 @@ async function run() {
         try {
             await exec.exec(command);
         } catch (err) {
-            core.setFailed('The ZAP Baseline scan has failed, starting to analyze the alerts. err: ' + err.toString());
+            if (err.toString().includes('exit code 3')) {
+                core.setFailed('failed to scan the target: ' + err.toString());
+                return
+            }
+
+            if ((err.toString().includes('exit code 2') || err.toString().includes('exit code 1'))
+                    && String(failAction).toLowerCase() === 'true') {
+                console.log(`[info] By default ZAP Docker container will fail if it identifies any alerts during the scan!`);
+                core.setFailed('Scan action failed as ZAP has identified alerts or failed to scan the target, ' +
```

Comment:
Remove `or failed to scan the target.`? That shouldn't happen (I think, per above check).